spec/site_github_munger_spec.rb

```diff
@@ -231,4 +231,27 @@
       end.to raise_error(Jekyll::GitHubMetadata::Client::BadCredentialsError)
     end
   end
+
+  context "render the 'uninject' fixture test site" do
+    let(:source) { File.expand_path("test-site-uninject", __dir__) }
+    let(:dest) { File.expand_path("../tmp/test-site-uninject-build", __dir__) }
+
+    it "process site twice (simulate reset), check API calls" do
+      config = Jekyll::Configuration.from({"source" => source, "destination" => dest})
+      site = Jekyll::Site.new(config)
+      site.process
+      expect_api_call "/repos/jekyll/github-metadata"
+      expect_api_call "/repos/jekyll/github-metadata/releases/latest"
+      expect_api_call "/orgs/jekyll"
+      site.process
+      expect_api_call "/repos/jekyll/github-metadata"
```

Comment:
A little more context could be helpful here. Could you add a comment along the lines of "We expect API calls again for the fields we use in the fixture site, but NOT for fields we don't"?